src/Resources/ansible/group_vars/deploy.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+#######
+# Dir #
+#######
+
+manala_deploy_dir: /srv/app
+
+############
+# Releases #
+############
+
+manala_deploy_releases: 5
+
+############
+# Strategy #
+############
+
+manala_deploy_strategy: git
+
+manala_deploy_strategy_git_repo:    "{{ lookup('pipe','git ls-remote --get-url origin') }}"
+manala_deploy_strategy_git_version: master
+
+##########
+# Copied #
+##########
+
+manala_deploy_copied:
+  - vendor
+  - node_modules
+
+##########
+# Shared #
+##########
+
+manala_deploy_shared_files:
+  - app/config/parameters.yml
+
+manala_deploy_shared_dirs:
+  - var/logs
+  - var/geoip2
```

Comment:
Je vais clean
(english would be greatly appreciated, c'est de l'open source 😆 )